lib/improver/utilities/cube_manipulation.py

```diff
@@ -238,7 +247,7 @@ def process(self, cubes_in):
         for concatenating cubes with differently numbered realizations).
```

Comment:
Might be worth adding something to the docstring to say what happens if a cube is passed in rather than a cubelist.

Reply:
Done here and for `MergeCubes.process`.